mp/src/game/shared/momentum/weapon/weapon_mom_smg.cpp

```diff
@@ -13,8 +13,8 @@ END_NETWORK_TABLE();
 BEGIN_PREDICTION_DATA(CMomentumSMG)
 END_PREDICTION_DATA();
 
-LINK_ENTITY_TO_CLASS(weapon_momentum_smg, CMomentumSMG);
-PRECACHE_WEAPON_REGISTER(weapon_momentum_smg);
+LINK_ENTITY_TO_CLASS(weapon_momentum_machinegun, CMomentumSMG);
```

Comment:
The SMG class can also be renamed to be just `MG`, same with the file name.

Reply:
Decided to go with the full `Machinegun` name, as this was the only weapon to be abbreviated now. Renamed the enums as well.